platformio.ini

```diff
@@ -8,14 +8,34 @@
 ; Please visit documentation for the other options and examples
 ; https://docs.platformio.org/page/projectconf.html
 
-[env:esp32-poe-iso]
-platform = espressif32 @ ^6.5.0
-board = esp32-poe-iso
-framework = arduino
-lib_deps = 
+; [env:esp32dev]
+; platform = espressif32 @ ^6.5.0
+; board = esp32-s2-saola-1
+; framework = arduino
+; lib_deps = 
+; 	me-no-dev/AsyncTCP @ ^1.1.1
+; 	https://github.com/me-no-dev/ESPAsyncWebServer.git#f71e3d427b5be9791a8a2c93cf8079792c3a9a26
+;     https://github.com/petchmakes/IRremoteESP8266.git#9630be3
+; 	bblanchon/ArduinoJson@^7.0
+; monitor_speed = 115200
+[platformio]
+default_envs = esp32-poe-iso
+
+[common]
+lib_deps_external  = 
 	me-no-dev/AsyncTCP @ ^1.1.1
 	https://github.com/me-no-dev/ESPAsyncWebServer.git#f71e3d427b5be9791a8a2c93cf8079792c3a9a26
-	https://github.com/petchmakes/IRremoteESP8266.git#9630be3
-	pedroalbuquerque/ESP32WebServer@^1.0
-	bblanchon/ArduinoJson@^7.0.2
+    https://github.com/petchmakes/IRremoteESP8266.git#9630be3
+	bblanchon/ArduinoJson@^7.0
 monitor_speed = 115200
+
+[env:esp32wroom]
+board = az-delivery-devkit-v4
+platform = espressif32 @ ^6.5.0
+framework = arduino
+
+[env:esp32-poe-iso]
+board = esp32-poe-iso
+platform = espressif32 @ ^6.5.0
+framework = arduino
+build_flags = -DBOARD_OLIMEX_FLAG=1
```

Comment:
Was more thinking you could set ``-DBLASTER_PIN_IR_INTERNAL=GPIO_NUM4``, etc here.

Reply:
You're right, my C/C++ skills are  rusty...